src/main/java/com/teamc/fms/entity/Tenant.java

```diff
@@ -0,0 +1,36 @@
+package com.teamc.fms.entity;
+
+import jakarta.persistence.*;
+import jakarta.validation.constraints.NotNull;
+import lombok.*;
+
+import java.time.LocalDate;
+
+@Entity
+@Table(name = "tenant")
+@Getter
+@Setter
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class Tenant {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.UUID)
+    private String id;
+    @NotNull
+    private String firstName;
+    @NotNull
+    private String lastName;
+    @NotNull
+    private String email;
+    @NotNull
+    private String phoneCode;
+    @NotNull
+    private String phoneNumber;
+    @NotNull
+    @ManyToOne
+    private Country nationality;
+    @NotNull
+    private LocalDate birthDate;
```

Comment:
phonceCode, phoneNumber, nationality and birthdate are nullable
![image](https://github.com/web-development-course-way/fms/assets/31379298/2b8d01b9-3245-4730-a796-d62f87b7c22b)